### PR TITLE
Added initial scale for better viewing

### PR DESCRIPTION
**Pull request by uttkarsh-cloud**
Helps in better viewing

### CODE REVIEW

**Comment by leemunroe** [comment]
Thank you! Can you provide a before/after screenshot to see what this does exactly?

**Comment by uttkarsh-cloud** [comment]
Ya you can see by going through the pull request it just sets the zoom level according to the device

File: email.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <!doctype html>
 <html>
   <head>
-    <meta name="viewport" content="width=device-width" />
+    <meta name="viewport" content="width=device-width" initial-scale=1.0/>
```
Comment (Malvoz) [inline email.html:4]: `initial-scale` needs to be a comma separated value of the `content` attribute.

**Comment by webketje** [comment]
@leemunroe 
> The initial-scale property controls the zoom level when the page is first loaded. [...] many browsers can display their pages in a smaller physical size by translating multiple hardware pixels for each CSS "pixel". [...] On high dpi screens, pages with initial-scale=1 will effectively be zoomed by browsers.
https://developer.mozilla.org/en-US/docs/Web/HTML/Viewport_meta_tag

File: email.html (hunk 1 above)
Comment (webketje) [reply email.html:4]: needs to be `<meta name="viewport" content="width=device-width, initial-scale=1.0"/>

Comment (leemunroe) [reply email.html:4]: Gotcha. @uttkarsh-cloud could you make this change? If you have a before and after screenshot as well I would love to understand what the visual difference is (I tested it myself and don't see anything, so wondering what I'm missing).

**Comment by webketje** [comment]
[Without initial-scale](https://htmlpreview.github.io/?https://gist.github.com/webketje/fe46d80d1c3f8168e8387988ddb25f5e/raw/b38bdc359a9b123562d437a94843c96566dfa802/without-initial-scale.html)
[With initial-scale=1.0](https://htmlpreview.github.io/?https://gist.github.com/webketje/fe46d80d1c3f8168e8387988ddb25f5e/raw/b38bdc359a9b123562d437a94843c96566dfa802/with-initial-scale.html)

**Comment by leemunroe** [comment]
Thanks although I was looking for an email client screenshot. I'd like to understand what this changes in an email and if it is needed.

**Comment by Karaca-star** [comment]
dude i don't understand you


30 Eyl 2021 Per 06:50 tarihinde Lee Munroe ***@***.***> şunu
yazdı:

**Comment by webketje** [comment]
> Thanks although I was looking for an email client screenshot. I'd like to understand what this changes in an email and if it is needed.

Web mail clients are also email clients. In this article 2 GIFS are included with the difference in font-size visible: https://css-tricks.com/probably-use-initial-scale1/

I also did a simple test with `email-inlined.html` on Gmail Web mobile in Firefox on a Samsung S9, portrait.
These are the results:

<details>
<summary>Screenshot without initial-scale:1</summary>
Without initial-scale:1
</details>

<details>
<summary>Screenshot with initial-scale:1</summary>
With initial-scale:1
</details>

*Note: I did have to change the `width: 580px;` style rule on `.container` to be `width: 100%;` (if you use width=device-width and the pixel-specified width is bigger than the viewport, the content will overflow)

On the Gmail mobile APP, both work as expected.

**Comment by leemunroe** [comment]
@webketje Awesome, thanks for sharing. I was missing that this fix was specifically for web mail clients.

**Pull request closed without merging**